shared/team-building/service-tab-bar.desktop.tsx

```diff
@@ -237,6 +244,9 @@ const styles = Styles.styleSheetCreate(() => ({
   tabBarContainer: {
     minHeight: 30,
   },
+  wonderland: {
+    color: Styles.globalColors.white,
```

Comment:
?

Reply:
The rabbit appears transparent with the default transparent text color. This makes it full opacity.